grails-app/views/mainContainer/index.gsp

```diff
@@ -1,8 +1,15 @@
 <html>
   <head>
+
+  <g:if test ="${openUrl}">
+    <meta name="layout" content="unguarded_component"/>
+  </g:if>
```

Comment:
I don't like conditional layout rendering this way. Conditional content, yes, but layouts are more brittle this way. This index page should be agnostic about *how* it is invoked. If there needs to be a different kind of layout, then it should be in a different gsp. Better to have a separate gsp that uses this layout.